webroot/api/notices/index.php

```diff
@@ -11,14 +11,15 @@
 }
 
 $notices = $SQL->getNotices();
+$jsonArray = [];
 foreach ($notices as $notice) {
-    echo $notice["title"] . "\r\n";
-    echo date('m-d-Y', strtotime($notice["date"])) . "\r\n";
-
-    $lineArr = explode("\r\n", wordwrap($notice["message"], $CHAR_WRAP));
-    foreach ($lineArr as $line) {
-        echo $line;
-    }
-
-    echo "\r\n\r\n";
+    $formattedNotice = [
+        "title" => $notice["title"],
+        "date" => date('m-d-Y', strtotime($notice["date"])),
+        "message" => wordwrap($notice["message"], $CHAR_WRAP)
```

Comment:
We don't want to wordwrap in JSON output since that will be handled by whatever is calling the API. Just add the message body directly here.
```suggestion
        "message" => $notice["message"]
```

Also remove any entries of `$CHAR_WRAP` from the rest of the file.